package.json

```diff
@@ -1,22 +1,18 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
   "funding": "https://opencollective.com/turf",
   "scripts": {
     "lint": "npm-run-all lint:*",
     "lint:eslint": "eslint packages",
     "lint:prettier": "prettier --check .",
-    "lint:mrl": "mrl check",
-    "lint:escheck-require": "es-check es8 packages/*/dist/js/index.js packages/turf/turf.min.js",
-    "lint:escheck-esm": "es-check --module es8 packages/*/dist/es/index.js",
-    "lint:escheck-es5": "es-check es5 packages/turf/turf.min.js",
-    "postlint": "documentation lint packages/turf-*/index.js",
-    "prepare": "lerna run build && node ./scripts/add-import-extensions.js",
-    "pretest": "npm run lint",
-    "test": "lerna run test",
-    "posttest": "lerna run --scope @turf/turf last-checks",
+    "lintDISABLED:mrl": "mrl check",
+    "lint:escheck-cjs": "es-check es8 packages/*/dist/cjs/index.cjs packages/turf/turf.min.js",
+    "lint:escheck-esm": "es-check --module es8 packages/*/dist/esm/index.mjs",
+    "lint:escheck-web": "es-check es5 packages/turf/turf.min.js",
+    "lint:docs": "documentation lint packages/turf-*/index.js packages/turf-*/index.mjs",
```

Comment:
Does this work with .ts modules?

Reply:
You mean "documentation lint"? Good question. I'll check and we can include turf-*/index.ts as a seperate change.